packages/lit-element/src/test/platform-support/lit-element-bundle_test.html

```diff
@@ -0,0 +1,178 @@
+<html>
+  <head>
+    <link rel="stylesheet" href="../../../node_modules/mocha/mocha.css" />
+    <!-- Loads polyfills but does not force settings... -->
+    <meta name="manual-polyfills" />
+    <script src="../../../node_modules/@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
+    <script src="../../../platform-support.js"></script>
+    <script src="../../../lit.min.js"></script>
+  </head>
+
+  <body>
+    <script type="module">
+      // TODO(sorvell): node resolution doesn't seem to be working here.
+      import {
+        mocha,
+        sessionFinished,
+        sessionFailed,
+      } from '../../../../tests/node_modules/@web/test-runner-mocha/dist/standalone.js';
+      import {assert} from '../../../node_modules/@esm-bundle/chai/esm/chai.js';
+      import {
+        getComputedStyleValue,
+        generateElementName,
+        nextFrame,
+      } from '../test-helpers.js';
+      // import { mocha, sessionFinished, sessionFailed } from '@web/test-runner-mocha';
+
+      (async () => {
```

Comment:
Doesn't seem like the async iffe is required?

Reply:
Done.